pyaedt/modules/Boundary.py

```diff
@@ -3350,3 +3351,738 @@ def delete(self):
+        new_node = self._Node(name, self._app, node_type="InternalNode", props=props_dict, network=self)
+        self._nodes.append(new_node)
+        self._add_to_props(new_node)
+        return new_node
+
+    @pyaedt_function_handler()
+    def add_boundary_node(self, name, assignment_type, value):
+        """
+
+        Parameters
+        ----------
+        name: str
+            String containing the name of the node.
+        assignment_type: str
+            String containing the type assignment. It can be ``"Power"`` or ``"Temperature"``.
+        value: str or float or dict
+            String or float or dict containing the value of the assignment. If a float is passed ``"W"``
+             or ``"cel"`` is used according to ``assignment_type``. If ``type`` is ``"Power"`, a
+             dictionary can be passed to use temperature dependent or transient assignment.
+
+        Returns
+        -------
+        bool
+            True if successful.
+
+        Examples
+        --------
+
+        >>> import pyaedt
+        >>> app = pyaedt.Icepak()
+        >>> network = pyaedt.modules.Boundary.Network(app)
+        >>> network.add_boundary_node("TestNode", "Temperature", 2)
+        >>> ds = app.create_dataset1d_design("Test_DataSet", [1, 2, 3], [3, 4, 5])
+        >>> network.add_boundary_node("TestNode", "Power", {"Type": "Temp Dep",
+        >>>                                                       "Function": "Piecewise Linear",
+        >>>                                                       "Values": "Test_DataSet"})
+        """
+        if assignment_type not in ["Power", "Temperature"]:
+            raise AttributeError('``type`` can be only ``"Power"`` or ``"Temperature"``')
+        if isinstance(value, (float, int)):
+            if assignment_type == "Power":
+                value = str(value) + "W"
+            else:
+                value = str(value) + "cel"
+        if isinstance(value, dict) and assignment_type == "Temperature":
+            raise AttributeError(
+                "Temperature dependent or transient assignment is not supported in a temperature boundary node."
+            )
+        new_node = self._Node(
+            name,
+            self._app,
+            node_type="BoundaryNode",
+            props={"ValueType": assignment_type + "Value", assignment_type: value},
+            network=self,
+        )
+        self._nodes.append(new_node)
+        self._add_to_props(new_node)
+        return new_node
+
+    @pyaedt_function_handler()
+    def _add_to_props(self, new_node, type_dict="Nodes"):
+        try:
+            self.props[type_dict].update({new_node.name: new_node.props})
+        except:
+            self.props[type_dict] = {new_node.name: new_node.props}
+
+    @pyaedt_function_handler()
+    def add_face_node(
+        self, face_id, name=None, thermal_resistance=None, material=None, thickness=None, resistance=None
+    ):
+        """
+
+        Parameters
+        ----------
```

Comment:
I understand that this is a work in progress. Because the data type is described in the same line as the parameter name is given, you do not need (or want) to include it again in the description. For example, instead of "String with the material specification ....)" use "Material specification ....". Please follow the guidance given in earlier suggestions for improvements as you finalize this content.